Genome/Source/Foundation/Foundation.swift

```diff
@@ -8,6 +8,7 @@
 //  MIT
 
 import PureJsonSerializer
+import Foundation
```

Comment:
Did no research why, but without this import the OS X framework will fail during linking